api/src/application/client/client-service.js

```diff
@@ -88,10 +88,15 @@ module.exports = (bookshelf) => ({
       .fetch({ withRelated: clientRelationships });
   },
 
+  findRedirectUriByClientId(clientId) {
```

Comment:
Clients can have many redirect uris configured. You won't be able to make a decision about which redirect uri to use based solely on the clientId. You need to update the invite and resent-invite routes to expect a `redirect_uri` body parameter so that the app can tell OIDC _which_ redirect uri to use.

You must then validate that the redirect uri provided matches one of the redirect_uris configured for the client